packages/asset-swapper/src/utils/market_operation_utils/aave_reserves_cache.ts

```diff
@@ -0,0 +1,115 @@
+import { logUtils } from '@0x/utils';
+import { gql, request } from 'graphql-request';
+
+import { constants } from '../../constants';
+
+const RESERVES_GQL_QUERY = gql`
+    {
+        reserves(where: { isActive: true, isFrozen: false }, orderBy: totalLiquidity, orderDirection: desc) {
+            id
+            name
+            isActive
+            isFrozen
+            underlyingAsset
+            totalLiquidity
+            aToken {
+                id
+                underlyingAssetAddress
+            }
+            pool {
+                id
+                lendingPool
+            }
+            decimals
+            symbol
+        }
```

Comment:
We don't actually use a lot of these? I think you can leave out fields even if they're used in the `where` and the `orderBy` clauses. `id`s are also optional afaik

Reply:
Yeah, I used some of this extra info for debugging during implementation, and grabbing id's everywhere is an old habit, because caching in heavier GQL clients like Apollo break if you don't fetch ids :feelsgood: 

Will clean it up 👍

Reply:
Removed most of the unused fields from the data fetching and caching. Prefer to keep the ids tho as that is good GQL hygiene, even if our GQL client doesn't do any fancy query deduping/batching or response catching 😄